content/en/docs/ibm/deploy/install-kubeflow-on-IBM-openshift.md

```diff
@@ -52,15 +52,22 @@ export KF_DIR=${BASE_DIR}/${KF_NAME}
 
 # Set the configuration file to use, such as:
 export CONFIG_FILE=kfctl_ibm.yaml
-export CONFIG_URI="https://raw.githubusercontent.com/kubeflow/manifests/master/kfdef/kfctl_openshift.master.kfptekton.yaml"
+export CONFIG_URI="https://raw.githubusercontent.com/kubeflow/manifests/master/distributions/kfdef/kfctl_openshift.master.kfptekton.yaml"
 
 # Generate Kubeflow:
 mkdir -p ${KF_DIR}
 cd ${KF_DIR}
-curl -L ${CONFIG_URI} > ${CONFIG_FILE}
+
+wget ${CONFIG_URI} -O kfctl.yaml
```

Comment:
should be change kfctl.yaml to `${CONFIG_FILE}`?

Reply:
making that change and pushing again.